### DISCUSSION TITLE
[WASM] House keeping on Fonts.css after upgrade to UNO 4.7

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#11076] [Q&A] [answered]
Previously I had the sections below in my Fonts.css file. I suppose they are legacy now (see ['Shared fonts' section here](https://platform.uno/blog/uno-platform-4-7-new-project-template-performance-improvements-and-more/)) - and initial tests are looking good. 

However, I wanted to be sure: could I savely remove those sections?
```
/* Workaround for uno issue https://github.com/unoplatform/uno/issues/693 */
body::before {
    font-family: "Roboto", sans-serif;
    background: transparent;
    content: "";
    opacity: 0;
    pointer-events: none;
    position: absolute;
}

/* https://github.com/unoplatform/uno/issues/4304 */
@font-face {
    font-family: 'Segoe UI';
    src: local('system-ui'), local('Segoe UI'), local('-apple-system'), local('BlinkMacSystemFont'), local('Inter'), local('Cantarell'), local('Ubuntu'), local('Roboto'), local('Open Sans'), local('Noto Sans'), local('Helvetica Neue'), local('sans-serif');
}```

**Comment by jeromelaban** [comment, marked as answer]
You can safely remove the first section, indeed.

The second one is different, as it relates to default fonts (https://github.com/unoplatform/uno/issues/3954) and is not fixed yet we'll adjust the link in the CSS which is incorrect. Thanks!

**Reply by DierkDroth** [reply]
Thanks @jeromelaban 

On the second one: is there anything I should change in my Fonts.css or should I just leave 'as is'?

**Reply by jeromelaban** [reply]
You can leave it as-is.
